src/composables/maskeditor/useBrushDrawing.ts

```
@@ -2,6 +2,7 @@
 import { ref, watch, nextTick, onUnmounted } from 'vue'
 import QuickLRU from '@alloc/quick-lru'
 import { debounce } from 'es-toolkit/compat'
+import { tgpu } from 'typegpu'
```

ESLint

[error] Unable to resolve path to module 'typegpu'.

(import-x/no-unresolved)